lib/spack/spack/package.py

```diff
@@ -410,6 +422,13 @@ class PackageBase(with_metaclass(PackageMeta, PackageViewMixin, object)):
     # These are default values for instance variables.
     #
 
+    # Automatically detecting packages with code cannot be based on the
```

Comment:
IMO this is too detailed an explanation for this attribute. I think something like

> Whether this package has code; packages without their own code might be used to install a group of other Spack packages

would be more useful here.

Reply:
Changed.